src/wp-includes/user.php

```diff
@@ -3012,19 +3012,57 @@ function retrieve_password( $user_login = null ) {
 	 */
 	$message = apply_filters( 'retrieve_password_message', $message, $key, $user_login, $user_data );
 
+	// Wrap the single notification email arguments in an array to pass them to the retrieve_password_notification_email filter.
+	$retrieve_password_notification_email = array(
+		'to'      => $user_email,
+		'subject' => $title,
+		'message' => $message,
+		'headers' => '',
+	);
+
+	/**
+	 * Filters the contents of the reset password notification email sent to the user.
+	 *
+	 * @since x.x.x
```

Comment:
You can directly use 6.0.0 since the ticket was milestoned to WP 6.0 :)

Reply:
Done!